src/Inventory/Asset/MainAsset.php

```diff
@@ -595,8 +609,10 @@ public function rulepassed($items_id, $itemtype, $rules_id, $ports_id = 0)
         $val->is_dynamic = 1;
         $val->entities_id = $entities_id;
 
-        $val->states_id = $this->states_id_default ?? $this->item->fields['states_id'] ?? 0;
```

Comment:
Are default configured values automatically used already?

Reply:
you're right, missed these